src/problem2.kt

```diff
@@ -0,0 +1,12 @@
+fun fibonacci(): Sequence<Int> =
```

Comment:
I think the return value and type can be inferred here?

``` fun fibonacci() = generateSequence(Pair(0, 1)) { Pair(it.second, it.first + it.second) }.map { it.first } ```

Reply:
good one, I'll change that.